apps/web/src/screens/Profile/modals/TransactionSuccessful/index.tsx

```diff
@@ -46,13 +74,15 @@ const TransactionSuccessfulScreen = () => {
 					backgroundColor="#56667433"
 				/>
 
-				<Stack paddingHorizontal={36}>
+				<Stack>
 					<InfoWrapper>
 						<Stack padding={16}>
 							<WalletInfo
-								networkLogo={networkLogo}
-								networkName={networkName}
-								walletAddress={walletAddress}
+								networkLogo={networkData?.icon || ''}
+								networkName={networkData?.name || ''}
```

Comment:
if we can make sure `icon` and `name` existed, use `networkData?.icon as string`.